### PR TITLE
MLDB-1186 make default sql builtins functions available in column expr

### PR DESCRIPTION
**Pull request by steadtler**

### CODE REVIEW

File: sql/binding_contexts.cc [hunk 1]
```diff
@@ -169,13 +169,24 @@ doGetFunction(const Utf8String & tableName,
...
+         return {[=] (const std::vector<ExpressionValue> & args,
                  const SqlRowScope & context)
             {
                 auto & col = static_cast<const ColumnContext &>(context);
                 return fn(col.columnName, args);
             },
             std::make_shared<Utf8StringValueInfo>()};
+    }
+
+    auto sqlfn = SqlBindingScope::doGetFunction(tableName, functionName, args);
```
Comment (guyd) [inline sql/binding_contexts.cc]: Will this break training code like svd which do not support operation on columns but support column expression?

Comment (guyd) [inline sql/binding_contexts.cc]: After discussion with Mathieu, some check will have to be written to ensure that the training does not fail badly on column expression with functions.  I will do that as part of MLDB-1104.

**Comment by guyd** [comment]
Mathieu, are you planning to add the test case you mentioned to me?

**Comment by steadtler** [comment]
@guyd le vrai test ca va être Mich! J'ai ajouté un test de base en attendant.

**Comment by guyd** [comment]
+1

**Pull request merged**